.github/workflows/test_pr.yml

```diff
@@ -20,6 +20,13 @@ jobs:
       fail-fast: false
       matrix:
         platform: [ "windows-latest", "ubuntu-latest", "macos-latest" ]
+        include:
+          - platform: "windows-latest"
+            vcpkg_triplet: "x64-windows-static"
```

Comment:
I had to manually select compiler triplets because of the way blosc is configured on vcpkg. The type of target library (static/shared) is encoded in the name of the library (i.e., blosc_static). Ubuntu and MacOS, by default, will compile blosc as a static library and Windows as a shared library. By selecting the triplet for Windows manually (`x64-windows-static`), we're ensuring that the output is consistent.

I may submit a patch to vcpkg. This is annoying.

Reply:
That's interesting, do you need triplets for all three platforms?